code/datums/components/rotation.dm

```diff
@@ -101,26 +94,23 @@
 /datum/component/simple_rotation/proc/ExamineMessage(datum/source, mob/user, list/examine_list)
 	SIGNAL_HANDLER
 
-	if(rotation_flags & ROTATION_ALTCLICK)
-		examine_list += span_notice("Alt-click to rotate it clockwise.")
+	examine_list += span_notice("Alt + Right-click to rotate it clockwise. Alt + Left-click to rotate it counterclockwise.")
+	if(rotation_flags & ROTATION_REQUIRE_WRENCH)
+		examine_list += span_notice("This requires a wrench to be rotated.")
 
-/datum/component/simple_rotation/proc/HandRot(datum/source, mob/user, rotation = default_rotation_direction)
+/datum/component/simple_rotation/proc/RotateRight(datum/source, mob/user)
```

Comment:
Can you change the procs you create or rename to use `snake_case()`? We have an unholy mix in the codebase and not much of a standard, though we seem to prefer snake case in variables at least.

Reply:
Ideally you would want to use different styles to identify different things.  We already use ALL_CAPS for defines that are almost always constants.  If `snake_case` is used for vars, then we should probably use `camelCase` for procs.  Enforcing this would be difficult though and the size of the codebase would require a lot of tedious code rewriting.  But keeping `snake_case` for both vars and procs to me seems like a waste.

Reply:
The vast majority of the procs already use `camel_case`, the second most used being `PascalCase`.
I suppose that is because the native BYOND procs use either, without much distinction.
`camelCase` is not that used here.
I'd stick with what's most used, really, but I get where you're coming.
At the end of the day this is all rather subjective and you are free not to make the change.

Reply:
Personally I've never found there to be a distinction between `PascalCase` vs `camelCase` and consider them to be the same thing.  I agree that when in doubt to just go with whatever the main language uses as a standard.  With that in mind, I'll go ahead and change the proc names to Pascal then.

Reply:
https://github.com/tgstation/tgstation/pull/64441/commits/19445943376e90892dc7ee56393570b2e8e62213